dev-python/pyilmbase/files/pyilmbase-2.3.0-link-pyimath.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/PyImath/Makefile.am b/PyImath/Makefile.am
+index bf87542..44f0dc3 100644
+--- a/PyImath/Makefile.am
++++ b/PyImath/Makefile.am
+@@ -82,7 +82,7 @@ imathmodule_la_SOURCES = imathmodule.cpp \
+     PyImathBasicTypes.cpp 
+ 
+ imathmodule_la_LDFLAGS = -avoid-version -module
+-imathmodule_la_LIBADD  = -lPyImath @BOOST_PYTHON_LIBS@
++imathmodule_la_LIBADD  = $(lib_LTLIBRARIES) @BOOST_PYTHON_LIBS@
```

Comment:
I don't think it's a good idea to reference generic variable here. List the specific .la file it needs instead.

I'd also appreciate if you verified that other directories don't have similar LIBADDs with just-built `-lfoo`.